codegen.js

```diff
@@ -19,17 +22,17 @@ const fetchSchema = async () => {
   const data = await response.json();
   const schema = buildClientSchema(data.data);
 
-  return printSchema(schema);
+  writeFileSync(SCHEMA_LOCATION, printSchema(schema));
 };
 
 const codegen = async () => {
   try {
-    const schema = await fetchSchema();
+    await fetchSchema();
 
     const graphqlFiles = './src/graphql/**/*.graphql';
 
     generate({
-      schema,
+      schema: SCHEMA_LOCATION,
```

Comment:
This change was made as the string loader doesn't take big files (>64kb) and would have been broken at some point anyways.